packages/core/src/components/FormLabel/FormLabel.example.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/display-name */
+import FormLabel from './FormLabel';
+import React from 'react';
+
+export default function() {
+  return (
+    <FormLabel
```

Comment:
I replaced the `PropTypes.string` with
```js
function(props, propName, componentName) {
  if (props.errorMessage && props[propName] == null) {
    return new Error(`Prop \`${propName}\` is required when specifying an \`errorMessage\`. None specified for ${componentName}.`);
  }
  else {
    return PropTypes.node.apply(PropTypes.node, arguments);
  }
}
```
but it says `custom` in the doc page, which is unhelpful, isn't it?

Reply:
Hmm, we can punt on this..I don't want it to block merging this PR's actual feature. 

I believe `fieldId` would also be required if `props.component` is `label` (the default), since it would be used as the label's `for` attribute value. I agree having it say `custom` on the doc page isn't ideal, but I think that's solveable.

Reply:
What if we made it required, but if people really want to omit it, they could just say `fieldId=""`?

Reply:
@pwolfert I think I'm cool with that (as well as supporting `null`). The one case where it wouldn't be required is when the component is a `legend` (like in `ChoiceList`).